examples/cms-sanity/next.config.js

```diff
@@ -0,0 +1,5 @@
+require('dotenv').config()
+
+module.exports = {
+  env: {},
```

Comment:
Do we need `NEXT_EXAMPLE_CMS_DATOCMS_PREVIEW_SECRET` here?

Reply:
Not unless we want it inlined at build time instead of providing it in the environment during runtime